ci/si_pipeline.sh

```diff
@@ -70,10 +69,10 @@ case $CLUSTER_LAUNCH_CODE in
       cp -f "$DOT_SHAKEDOWN" "$HOME/.shakedown"
       (cd tests && make init test)
       SI_CODE=$?
-      # if [ ${SI_CODE} -gt 0 ]; then
-      #  download-diagnostics-bundle
-      # fi
-      ./dcos-launch delete
+      if [ ${SI_CODE} -gt 0 ]; then
+        download-diagnostics-bundle
+      fi
+      ./bin/dcos-launch delete
```

Comment:
What if `dcos-launch` is already available in your system? 

I would recommend here to just modify the PATH and append the `./bin` subfolder.

Reply:
So far this script is only run by the CI and it assumes a specific installation path for dcos-launch.